src/ErrorHandler.php

```diff
@@ -29,20 +29,20 @@ public function onError($code, $message, $file, $line)
         switch ($code) {
             case E_NOTICE:
             case E_USER_NOTICE:
-                $exc = new Errors\Notice($message, $file, $line, debug_backtrace());
+                $exc = new Errors\Notice($message, '', 0, debug_backtrace());
```

Comment:
Why does `Errors\Base` accept `$file` & `$line` then? As of this PR, [that](https://github.com/airbrake/phpbrake/blob/fix/error-handler-test/src/ErrorHandler.php#L72) is the only place where we use it. If you say it's better to use `debug_backtrace()`, then it seems like we can replace that, too (and get rid of unused parameters).